test/test-agent-stopped.js

```diff
@@ -21,9 +21,14 @@ var http = require('http');
 
 describe('test-agent-stopped', function() {
   var agent;
-  before(function() {
+  before(function(done) {
     delete process.env.GCLOUD_PROJECT;
     agent = require('..').start();
+    // Wait 300ms for agent to fail getting remote project id.
```

Comment:
Using `nock` should allow us to reduce the timeout without increasing the flakiness but it would be great to have a solution that didn't rely on timing.